packages/client/hmi-client/src/workflow/ops/model-from-code/tera-model-from-code.vue

```diff
@@ -168,145 +166,118 @@ const codeBlock = {
 const codeBlocks = ref([codeBlock]);
 
 onMounted(async () => {
-	const session = await newSession('beaker', 'Beaker');
-	jupyterSession.value = session;
-
-	session.kernelChanged.connect((_context, kernelInfo) => {
-		const kernel = kernelInfo.newValue;
-		if (kernel?.name === 'beaker') {
-			session.iopubMessage.connect(iopubMessageHandler);
-			setKernelContext();
+	try {
+		const jupyterContext = buildJupyterContext();
+		if (jupyterContext) {
+			await kernelManager.init('beaker', 'Beaker', jupyterContext);
 		}
-	});
+	} catch (error) {
+		console.error('Error initializing Jupyter session:', error);
```

Comment:
Please use `logger.error()` for this, and display a toast with a title and message explaining that the notebook cannot be initialized.

Reply:
done